lib/reporters/cli/index.js

```diff
@@ -334,6 +349,17 @@ _.assignIn(PostmanCLIReporter, {
             summaryTable.push([colors.green(label), util.prettyms(timings[`${key}Average`])]);
         });
 
+        timingPhases && _.forEach({
```

Comment:
Let's use `timingPhases` to show all the timings in the following order and remove the above `timings` variable usage.
```
    dns: Number,             // duration of DNS lookup
    tcp: Number,             // duration of TCP connection
    secureHandshake: Number, // duration of secure handshake
    firstByte: Number,       // duration of HTTP server response
    download: Number,        // duration of HTTP download
    total: Number            // duration entire HTTP round-trip
```
Ref: https://github.com/postmanlabs/postman-runtime/blob/develop/docs/history.md

Reply:
Ok, will change.

Reply:
We still keep run.transfers right? It is being used for showing total data received.

Reply:
Yup, `run.transfers` will be kept.